dvc/main.py

```diff
@@ -75,6 +86,9 @@ def main(argv=None):
         # so won't be reused by any other subsequent run anyway.
         clean_repos()
 
+    if ret != 0:
```

Comment:
not sure if it should be here or rather in `finally` block

Reply:
This place is fine.